### DISCUSSION TITLE
Browse and beatmap listing icons should be the same

### DISCUSSION THREAD

**Discussion by weiye97** [ppy/osu#20715] [Ideas]
Now the beatmap listing icon at the top and the browse icon on the home screen are different symbols.
When navigating I was looking for the browse icon on the top of the screen and couldn't find it, until realizing that the icons are different.
They should be changed so they are the same icon.
![Annotazione 2022-10-11 184012](https://user-images.githubusercontent.com/81234042/195150549-a192b4d3-4a47-4be0-bcc4-00e0542211da.png)

**Comment by peppy** [comment]
The funny part is that neither of these icons are the final version, if I recall.

I don't know about the future of the "browse" button (existing) on the main menu. Time will tell with that one.
